  INFO root [2019-09-10 10:05:54,947] - 
  INFO root [2019-09-10 10:05:54,948] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 10:05:54,949] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 10:05:54,949] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 10:05:54,950] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 10:05:54,976] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:59060 )
  INFO root [2019-09-10 10:05:54,976] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:59060, SOMAXCONN=5.
  INFO root [2019-09-10 10:05:54,977] - server-configs:
  INFO root [2019-09-10 10:05:54,977] - 	internalTcpAddr : 192.168.56.1:59060
  INFO root [2019-09-10 10:05:54,977] - 	componentID : 6000
  WARN root [2019-09-10 10:05:54,987] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 10:05:56,492] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 10604 ----
  INFO root [2019-09-10 10:05:56,507] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 10:05:56,507] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 10:05:56,508] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 10:05:56,909] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 10:05:56,909] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 10:06:04,051] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 10:06:04,138] - Components::findComponents: found logger, addr:192.168.56.1:59067
  INFO root [2019-09-10 10:06:04,139] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 10:06:04,139] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 10:06:04,141] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-10 10:06:05,243] - Forwarding logs to logger[192.168.56.1:59067/0/0/0]...
 ERROR root [2019-09-10 10:12:12,291] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=179820056239, lastReceivedTime=179781582176, inactivityExceptionPeriod=134887197)
  INFO root [2019-09-10 10:12:12,293] - The following logs sent to logger failed:
  WARN root [2019-09-10 10:12:12,293] - ==> [2019-09-10 10:12:12 289] TCPPacketReceiver::processPendingEvents(192.168.56.1:59067): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:12:12,293] - ==> [2019-09-10 10:12:12 291] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=179820056239, lastReceivedTime=179781582176, inactivityExceptionPeriod=134887197)
  WARN root [2019-09-10 10:12:12,385] - TCPPacketReceiver::processPendingEvents(192.168.56.1:59068): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:12:12,388] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=179820274140, lastReceivedTime=179781601499, inactivityExceptionPeriod=134887197)
  INFO root [2019-09-10 10:12:12,393] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 10:12:12,393] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 10:12:12,394] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 10:12:12,394] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 10:12:13,928] - 
  INFO root [2019-09-10 10:12:13,929] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 10:12:13,930] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 10:12:13,931] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 10:12:13,931] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 10:12:13,951] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:59225 )
  INFO root [2019-09-10 10:12:13,951] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:59225, SOMAXCONN=5.
  INFO root [2019-09-10 10:12:13,951] - server-configs:
  INFO root [2019-09-10 10:12:13,951] - 	internalTcpAddr : 192.168.56.1:59225
  INFO root [2019-09-10 10:12:13,951] - 	componentID : 6000
  WARN root [2019-09-10 10:12:13,953] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-10 10:12:14,293] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20756, errorno=10048
  INFO root [2019-09-10 10:12:14,301] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 5384 ----
  INFO root [2019-09-10 10:12:14,327] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 10:12:14,329] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 10:12:14,329] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 10:12:14,735] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 10:12:14,735] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 10:12:21,870] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 10:12:21,913] - Components::findComponents: found logger, addr:192.168.56.1:59229
  INFO root [2019-09-10 10:12:21,914] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 10:12:21,914] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 10:12:21,915] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 10:12:22,017] - Components::findComponents: found dbmgr, addr:192.168.56.1:59227
  INFO root [2019-09-10 10:12:22,017] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 10:12:22,017] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 10:12:22,018] - Components::findComponents: found dbmgr, addr:192.168.56.1:59227
  WARN root [2019-09-10 10:12:23,019] - Forwarding logs to logger[192.168.56.1:59229/0/0/0]...
 ERROR root [2019-09-10 10:16:20,694] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180378964314, lastReceivedTime=180364000308, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 10:16:20,702] - The following logs sent to logger failed:
  WARN root [2019-09-10 10:16:20,702] - ==> [2019-09-10 10:16:20 692] TCPPacketReceiver::processPendingEvents(192.168.56.1:59229): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:16:20,702] - ==> [2019-09-10 10:16:20 695] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180378964314, lastReceivedTime=180364000308, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 10:16:20,796] - TCPPacketReceiver::processPendingEvents(192.168.56.1:59227): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:16:20,805] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180379209854, lastReceivedTime=180363932993, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 10:16:20,807] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 10:16:20,807] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 10:16:20,807] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 10:16:20,807] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 10:16:22,529] - 
  INFO root [2019-09-10 10:16:22,538] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 10:16:22,539] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 10:16:22,539] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 10:16:22,539] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 10:16:22,615] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:59423 )
  INFO root [2019-09-10 10:16:22,616] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:59423, SOMAXCONN=5.
  INFO root [2019-09-10 10:16:22,616] - server-configs:
  INFO root [2019-09-10 10:16:22,616] - 	internalTcpAddr : 192.168.56.1:59423
  INFO root [2019-09-10 10:16:22,616] - 	componentID : 6000
  WARN root [2019-09-10 10:16:22,621] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 10:16:24,126] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 9428 ----
  INFO root [2019-09-10 10:16:24,128] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 10:16:24,128] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 10:16:24,128] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 10:16:24,541] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 10:16:24,542] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 10:16:31,669] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 10:16:31,738] - Components::findComponents: found logger, addr:192.168.56.1:59426
  INFO root [2019-09-10 10:16:31,739] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 10:16:31,739] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 10:16:31,740] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 10:16:31,843] - Components::findComponents: found dbmgr, addr:192.168.56.1:59421
  INFO root [2019-09-10 10:16:31,843] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 10:16:31,843] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 10:16:31,843] - Components::findComponents: found dbmgr, addr:192.168.56.1:59421
  WARN root [2019-09-10 10:16:32,844] - Forwarding logs to logger[192.168.56.1:59426/0/0/0]...
 ERROR root [2019-09-10 10:20:18,326] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180913637319, lastReceivedTime=180855807699, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 10:20:18,330] - The following logs sent to logger failed:
  WARN root [2019-09-10 10:20:18,330] - ==> [2019-09-10 10:20:18 313] TCPPacketReceiver::processPendingEvents(192.168.56.1:59426): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:20:18,330] - ==> [2019-09-10 10:20:18 327] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180913637319, lastReceivedTime=180855807699, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-10 10:20:18,479] - TCPPacketReceiver::processPendingEvents(192.168.56.1:59421): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:20:18,488] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=180914000435, lastReceivedTime=180855684685, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 10:20:18,496] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 10:20:18,501] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 10:20:18,501] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 10:20:18,502] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-10 10:20:18,820] - ThreadPool::destroy(): waiting for thread(2)[0x1da3320,0x1da3a08,], try=1
  INFO root [2019-09-10 10:34:34,234] - 
  INFO root [2019-09-10 10:34:34,241] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 10:34:34,242] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 10:34:34,242] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 10:34:34,242] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 10:34:34,302] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:59873 )
  INFO root [2019-09-10 10:34:34,303] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:59873, SOMAXCONN=5.
  INFO root [2019-09-10 10:34:34,303] - server-configs:
  INFO root [2019-09-10 10:34:34,303] - 	internalTcpAddr : 192.168.56.1:59873
  INFO root [2019-09-10 10:34:34,303] - 	componentID : 6000
  WARN root [2019-09-10 10:34:34,305] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 10:34:35,811] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15104 ----
  INFO root [2019-09-10 10:34:35,811] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 10:34:35,812] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 10:34:35,812] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 10:34:36,236] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 10:34:36,237] - Components::process(): Request for the process of identity...
  WARN root [2019-09-10 10:34:42,407] - TCPPacketReceiver::processPendingEvents(192.168.56.1:59891): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-10 10:34:43,414] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 10:34:43,506] - Components::findComponents: found logger, addr:192.168.56.1:59871
  INFO root [2019-09-10 10:34:43,506] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 10:34:43,507] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 10:34:43,508] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-10 10:34:44,611] - Forwarding logs to logger[192.168.56.1:59871/0/0/0]...
 ERROR root [2019-09-10 10:38:45,827] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183405514939, lastReceivedTime=183379478694, inactivityExceptionPeriod=134887085)
  INFO root [2019-09-10 10:38:45,829] - The following logs sent to logger failed:
  WARN root [2019-09-10 10:38:45,829] - ==> [2019-09-10 10:38:45 826] TCPPacketReceiver::processPendingEvents(192.168.56.1:59871): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:38:45,830] - ==> [2019-09-10 10:38:45 828] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183405514939, lastReceivedTime=183379478694, inactivityExceptionPeriod=134887085)
  WARN root [2019-09-10 10:38:45,935] - TCPPacketReceiver::processPendingEvents(192.168.56.1:59877): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:38:45,938] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=183405765920, lastReceivedTime=183379704297, inactivityExceptionPeriod=134887085)
  INFO root [2019-09-10 10:38:45,943] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 10:38:45,945] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 10:38:45,945] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 10:38:45,945] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 10:56:50,325] - 
  INFO root [2019-09-10 10:56:50,328] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 10:56:50,330] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 10:56:50,330] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 10:56:50,330] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 10:56:50,380] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:60159 )
  INFO root [2019-09-10 10:56:50,380] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:60159, SOMAXCONN=5.
  INFO root [2019-09-10 10:56:50,380] - server-configs:
  INFO root [2019-09-10 10:56:50,380] - 	internalTcpAddr : 192.168.56.1:60159
  INFO root [2019-09-10 10:56:50,381] - 	componentID : 6000
  WARN root [2019-09-10 10:56:50,382] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-10 10:56:50,723] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20768, errorno=10048
  INFO root [2019-09-10 10:56:50,727] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 9176 ----
  INFO root [2019-09-10 10:56:50,727] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 10:56:50,728] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 10:56:50,728] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 10:56:51,149] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 10:56:51,161] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 10:56:58,329] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 10:56:58,349] - Components::findComponents: found logger, addr:192.168.56.1:60158
  INFO root [2019-09-10 10:56:58,349] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 10:56:58,349] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 10:56:58,351] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 10:56:58,454] - Components::findComponents: found dbmgr, addr:192.168.56.1:60157
  INFO root [2019-09-10 10:56:58,454] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 10:56:58,455] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 10:56:58,455] - Components::findComponents: found dbmgr, addr:192.168.56.1:60157
  WARN root [2019-09-10 10:56:59,455] - Forwarding logs to logger[192.168.56.1:60158/0/0/0]...
 ERROR root [2019-09-10 10:58:26,269] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=186061511532, lastReceivedTime=186048246624, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 10:58:26,275] - The following logs sent to logger failed:
  WARN root [2019-09-10 10:58:26,275] - ==> [2019-09-10 10:58:26 266] TCPPacketReceiver::processPendingEvents(192.168.56.1:60158): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:58:26,275] - ==> [2019-09-10 10:58:26 270] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=186061511532, lastReceivedTime=186048246624, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 10:58:26,381] - TCPPacketReceiver::processPendingEvents(192.168.56.1:60157): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 10:58:26,387] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=186061775814, lastReceivedTime=186048250917, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 10:58:26,410] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 10:58:26,417] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 10:58:26,418] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 10:58:26,418] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 11:01:10,337] - 
  INFO root [2019-09-10 11:01:10,354] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 11:01:10,355] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 11:01:10,356] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 11:01:10,356] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 11:01:10,399] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:60300 )
  INFO root [2019-09-10 11:01:10,400] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:60300, SOMAXCONN=5.
  INFO root [2019-09-10 11:01:10,400] - server-configs:
  INFO root [2019-09-10 11:01:10,400] - 	internalTcpAddr : 192.168.56.1:60300
  INFO root [2019-09-10 11:01:10,400] - 	componentID : 6000
  WARN root [2019-09-10 11:01:10,400] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 11:01:11,906] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 14744 ----
  INFO root [2019-09-10 11:01:11,907] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 11:01:11,907] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 11:01:11,907] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 11:01:12,344] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 11:01:12,346] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 11:01:19,482] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 11:01:19,554] - Components::findComponents: found logger, addr:192.168.56.1:60299
  INFO root [2019-09-10 11:01:19,555] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:01:19,555] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:01:19,556] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 11:01:19,561] - Components::findComponents: found logger, addr:192.168.56.1:60299
  INFO root [2019-09-10 11:01:19,564] - Components::findComponents: found dbmgr, addr:192.168.56.1:60298
  INFO root [2019-09-10 11:01:19,564] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:01:19,564] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:01:19,664] - Components::findComponents: found dbmgr, addr:192.168.56.1:60298
  WARN root [2019-09-10 11:01:20,666] - Forwarding logs to logger[192.168.56.1:60299/0/0/0]...
 ERROR root [2019-09-10 11:06:07,445] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=187099157963, lastReceivedTime=187038255983, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 11:06:07,447] - The following logs sent to logger failed:
  WARN root [2019-09-10 11:06:07,448] - ==> [2019-09-10 11:06:07 444] TCPPacketReceiver::processPendingEvents(192.168.56.1:60299): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:06:07,448] - ==> [2019-09-10 11:06:07 446] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=187099157963, lastReceivedTime=187038255983, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 11:06:07,551] - TCPPacketReceiver::processPendingEvents(192.168.56.1:60298): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:06:07,558] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=187099410621, lastReceivedTime=187038221735, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 11:06:07,575] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 11:06:07,577] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 11:06:07,578] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 11:06:07,578] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 11:06:09,314] - 
  INFO root [2019-09-10 11:06:09,330] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 11:06:09,333] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 11:06:09,333] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 11:06:09,333] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 11:06:09,389] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:60383 )
  INFO root [2019-09-10 11:06:09,390] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:60383, SOMAXCONN=5.
  INFO root [2019-09-10 11:06:09,390] - server-configs:
  INFO root [2019-09-10 11:06:09,390] - 	internalTcpAddr : 192.168.56.1:60383
  INFO root [2019-09-10 11:06:09,390] - 	componentID : 6000
  WARN root [2019-09-10 11:06:09,391] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 11:06:10,895] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12004 ----
  INFO root [2019-09-10 11:06:10,896] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 11:06:10,896] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 11:06:10,896] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 11:06:11,323] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 11:06:11,323] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 11:06:18,468] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 11:06:18,560] - Components::findComponents: found logger, addr:192.168.56.1:60381
  INFO root [2019-09-10 11:06:18,561] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:06:18,561] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:06:18,562] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 11:06:18,567] - Components::findComponents: found logger, addr:192.168.56.1:60381
  INFO root [2019-09-10 11:06:18,570] - Components::findComponents: found dbmgr, addr:192.168.56.1:60382
  INFO root [2019-09-10 11:06:18,571] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:06:18,571] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:06:18,672] - Components::findComponents: found dbmgr, addr:192.168.56.1:60382
  WARN root [2019-09-10 11:06:19,672] - Forwarding logs to logger[192.168.56.1:60381/0/0/0]...
 ERROR root [2019-09-10 11:07:51,456] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=187333182219, lastReceivedTime=187305980445, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 11:07:51,461] - The following logs sent to logger failed:
  WARN root [2019-09-10 11:07:51,461] - ==> [2019-09-10 11:07:51 452] TCPPacketReceiver::processPendingEvents(192.168.56.1:60381): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:07:51,462] - ==> [2019-09-10 11:07:51 457] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=187333182219, lastReceivedTime=187305980445, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 11:07:51,575] - TCPPacketReceiver::processPendingEvents(192.168.56.1:60382): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:07:51,578] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=187333456853, lastReceivedTime=187306005435, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 11:07:51,589] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 11:07:51,594] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 11:07:51,595] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 11:07:51,596] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-10 11:07:51,897] - ThreadPool::destroy(): waiting for thread(2)[0xfb5688,0xfb5210,], try=1
  INFO root [2019-09-10 11:07:53,551] - 
  INFO root [2019-09-10 11:07:53,557] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 11:07:53,559] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 11:07:53,559] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 11:07:53,560] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 11:07:53,694] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:60438 )
  INFO root [2019-09-10 11:07:53,694] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:60438, SOMAXCONN=5.
  INFO root [2019-09-10 11:07:53,694] - server-configs:
  INFO root [2019-09-10 11:07:53,694] - 	internalTcpAddr : 192.168.56.1:60438
  INFO root [2019-09-10 11:07:53,695] - 	componentID : 6000
  WARN root [2019-09-10 11:07:53,700] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 11:07:55,211] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15408 ----
  INFO root [2019-09-10 11:07:55,212] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 11:07:55,213] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 11:07:55,213] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 11:07:55,648] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 11:07:55,648] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 11:08:02,795] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 11:08:02,817] - Components::findComponents: found logger, addr:192.168.56.1:60439
  INFO root [2019-09-10 11:08:02,817] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:08:02,817] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:08:02,819] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 11:08:02,922] - Components::findComponents: found dbmgr, addr:192.168.56.1:60437
  INFO root [2019-09-10 11:08:02,922] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:08:02,922] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:08:02,922] - Components::findComponents: found dbmgr, addr:192.168.56.1:60437
  WARN root [2019-09-10 11:08:03,922] - Forwarding logs to logger[192.168.56.1:60439/0/0/0]...
 ERROR root [2019-09-10 11:13:45,017] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=188128694068, lastReceivedTime=188080655320, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-10 11:13:45,022] - The following logs sent to logger failed:
  WARN root [2019-09-10 11:13:45,022] - ==> [2019-09-10 11:13:45 016] TCPPacketReceiver::processPendingEvents(192.168.56.1:60439): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:13:45,022] - ==> [2019-09-10 11:13:45 017] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=188128694068, lastReceivedTime=188080655320, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-10 11:13:45,120] - TCPPacketReceiver::processPendingEvents(192.168.56.1:60437): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:13:45,120] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=188128928312, lastReceivedTime=188080640008, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-10 11:13:45,123] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 11:13:45,123] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 11:13:45,123] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 11:13:45,124] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 11:13:46,880] - 
  INFO root [2019-09-10 11:13:46,889] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 11:13:46,890] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 11:13:46,890] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 11:13:46,890] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 11:13:46,965] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:60519 )
  INFO root [2019-09-10 11:13:46,965] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:60519, SOMAXCONN=5.
  INFO root [2019-09-10 11:13:46,965] - server-configs:
  INFO root [2019-09-10 11:13:46,965] - 	internalTcpAddr : 192.168.56.1:60519
  INFO root [2019-09-10 11:13:46,965] - 	componentID : 6000
  WARN root [2019-09-10 11:13:46,968] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 11:13:48,484] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 8876 ----
  INFO root [2019-09-10 11:13:48,488] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 11:13:48,489] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 11:13:48,490] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 11:13:49,060] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 11:13:49,061] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 11:13:56,212] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 11:13:56,243] - Components::findComponents: found logger, addr:192.168.56.1:60518
  INFO root [2019-09-10 11:13:56,243] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:13:56,243] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:13:56,244] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 11:13:56,347] - Components::findComponents: found dbmgr, addr:192.168.56.1:60520
  INFO root [2019-09-10 11:13:56,347] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:13:56,347] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:13:56,352] - Components::findComponents: found dbmgr, addr:192.168.56.1:60520
  WARN root [2019-09-10 11:13:57,354] - Forwarding logs to logger[192.168.56.1:60518/0/0/0]...
 ERROR root [2019-09-10 11:18:17,820] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=188742500631, lastReceivedTime=188740569875, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 11:18:17,826] - The following logs sent to logger failed:
  WARN root [2019-09-10 11:18:17,826] - ==> [2019-09-10 11:18:17 813] TCPPacketReceiver::processPendingEvents(192.168.56.1:60518): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:18:17,827] - ==> [2019-09-10 11:18:17 820] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=188742500631, lastReceivedTime=188740569875, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-10 11:18:17,935] - TCPPacketReceiver::processPendingEvents(192.168.56.1:60520): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:18:17,940] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=188742772581, lastReceivedTime=188740529603, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 11:18:17,958] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 11:18:17,964] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 11:18:17,965] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 11:18:17,965] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 11:18:19,933] - 
  INFO root [2019-09-10 11:18:19,934] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 11:18:19,935] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 11:18:19,935] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 11:18:19,935] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 11:18:19,964] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:60619 )
  INFO root [2019-09-10 11:18:19,964] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:60619, SOMAXCONN=5.
  INFO root [2019-09-10 11:18:19,964] - server-configs:
  INFO root [2019-09-10 11:18:19,964] - 	internalTcpAddr : 192.168.56.1:60619
  INFO root [2019-09-10 11:18:19,965] - 	componentID : 6000
  WARN root [2019-09-10 11:18:19,965] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 11:18:21,469] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13148 ----
  INFO root [2019-09-10 11:18:21,473] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 11:18:21,475] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 11:18:21,475] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 11:18:21,961] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 11:18:21,980] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 11:18:29,218] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 11:18:29,296] - Components::findComponents: found logger, addr:192.168.56.1:60617
  INFO root [2019-09-10 11:18:29,297] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:18:29,297] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:18:29,298] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 11:18:29,300] - Components::findComponents: found logger, addr:192.168.56.1:60617
  INFO root [2019-09-10 11:18:29,306] - Components::findComponents: found dbmgr, addr:192.168.56.1:60621
  INFO root [2019-09-10 11:18:29,306] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 11:18:29,306] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 11:18:29,409] - Components::findComponents: found dbmgr, addr:192.168.56.1:60621
  WARN root [2019-09-10 11:18:30,409] - Forwarding logs to logger[192.168.56.1:60617/0/0/0]...
 ERROR root [2019-09-10 11:21:30,324] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=189175637069, lastReceivedTime=189152245182, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-10 11:21:30,328] - The following logs sent to logger failed:
  WARN root [2019-09-10 11:21:30,328] - ==> [2019-09-10 11:21:30 324] TCPPacketReceiver::processPendingEvents(192.168.56.1:60617): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:21:30,328] - ==> [2019-09-10 11:21:30 325] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=189175637069, lastReceivedTime=189152245182, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-10 11:21:30,431] - TCPPacketReceiver::processPendingEvents(192.168.56.1:60621): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 11:21:30,433] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=189175880541, lastReceivedTime=189152617462, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-10 11:21:30,449] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 11:21:30,463] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 11:21:30,463] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 11:21:30,463] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 14:56:33,520] - 
  INFO root [2019-09-10 14:56:33,566] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 14:56:33,567] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 14:56:33,567] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 14:56:33,567] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 14:56:33,585] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:61714 )
  INFO root [2019-09-10 14:56:33,586] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:61714, SOMAXCONN=5.
  INFO root [2019-09-10 14:56:33,586] - server-configs:
  INFO root [2019-09-10 14:56:33,586] - 	internalTcpAddr : 192.168.56.1:61714
  INFO root [2019-09-10 14:56:33,586] - 	componentID : 6000
  WARN root [2019-09-10 14:56:33,587] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 14:56:35,093] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 1748 ----
  INFO root [2019-09-10 14:56:35,093] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 14:56:35,093] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 14:56:35,094] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 14:56:35,551] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 14:56:35,552] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 14:56:42,715] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 14:56:42,787] - Components::findComponents: found logger, addr:192.168.56.1:61711
  INFO root [2019-09-10 14:56:42,787] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 14:56:42,788] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 14:56:42,789] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-10 14:56:43,902] - Forwarding logs to logger[192.168.56.1:61711/0/0/0]...
 ERROR root [2019-09-10 14:59:39,050] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218635554454, lastReceivedTime=218623050287, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 14:59:39,053] - The following logs sent to logger failed:
  WARN root [2019-09-10 14:59:39,054] - ==> [2019-09-10 14:59:39 048] TCPPacketReceiver::processPendingEvents(192.168.56.1:61711): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 14:59:39,054] - ==> [2019-09-10 14:59:39 051] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218635554454, lastReceivedTime=218623050287, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 14:59:39,160] - TCPPacketReceiver::processPendingEvents(192.168.56.1:61715): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 14:59:39,160] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218635802822, lastReceivedTime=218623342142, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 14:59:39,163] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 14:59:39,165] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 14:59:39,165] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 14:59:39,166] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 14:59:41,099] - 
  INFO root [2019-09-10 14:59:41,103] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 14:59:41,106] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 14:59:41,106] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 14:59:41,106] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 14:59:41,135] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:61808 )
  INFO root [2019-09-10 14:59:41,135] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:61808, SOMAXCONN=5.
  INFO root [2019-09-10 14:59:41,135] - server-configs:
  INFO root [2019-09-10 14:59:41,136] - 	internalTcpAddr : 192.168.56.1:61808
  INFO root [2019-09-10 14:59:41,136] - 	componentID : 6000
  WARN root [2019-09-10 14:59:41,138] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-10 14:59:41,480] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20619, errorno=10048
  INFO root [2019-09-10 14:59:41,484] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 16012 ----
  INFO root [2019-09-10 14:59:41,486] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 14:59:41,492] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 14:59:41,492] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 14:59:41,955] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 14:59:41,956] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 16:30:32,954] - 
  INFO root [2019-09-10 16:30:32,974] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 16:30:32,975] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 16:30:32,975] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 16:30:32,976] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 16:30:33,019] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63321 )
  INFO root [2019-09-10 16:30:33,020] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63321, SOMAXCONN=5.
  INFO root [2019-09-10 16:30:33,020] - server-configs:
  INFO root [2019-09-10 16:30:33,020] - 	internalTcpAddr : 192.168.56.1:63321
  INFO root [2019-09-10 16:30:33,020] - 	componentID : 6000
  WARN root [2019-09-10 16:30:33,028] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 16:30:34,542] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 10828 ----
  INFO root [2019-09-10 16:30:34,543] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 16:30:34,543] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 16:30:34,544] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 16:30:34,938] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 16:30:34,939] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 16:30:42,065] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 16:30:42,083] - Components::findComponents: found logger, addr:192.168.56.1:63322
  INFO root [2019-09-10 16:30:42,083] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:30:42,083] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:30:42,085] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-10 16:30:43,188] - Forwarding logs to logger[192.168.56.1:63322/0/0/0]...
 ERROR root [2019-09-10 16:38:24,192] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=231960002787, lastReceivedTime=231912369927, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 16:38:24,196] - The following logs sent to logger failed:
  WARN root [2019-09-10 16:38:24,196] - ==> [2019-09-10 16:38:24 191] TCPPacketReceiver::processPendingEvents(192.168.56.1:63322): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:38:24,196] - ==> [2019-09-10 16:38:24 192] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=231960002787, lastReceivedTime=231912369927, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-10 16:38:24,291] - TCPPacketReceiver::processPendingEvents(192.168.56.1:63323): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:38:24,292] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=231960229036, lastReceivedTime=231912416058, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 16:38:24,293] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 16:38:24,293] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 16:38:24,293] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 16:38:24,294] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 16:41:45,031] - 
  INFO root [2019-09-10 16:41:45,044] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 16:41:45,045] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 16:41:45,046] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 16:41:45,046] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 16:41:45,090] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63444 )
  INFO root [2019-09-10 16:41:45,090] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63444, SOMAXCONN=5.
  INFO root [2019-09-10 16:41:45,091] - server-configs:
  INFO root [2019-09-10 16:41:45,091] - 	internalTcpAddr : 192.168.56.1:63444
  INFO root [2019-09-10 16:41:45,091] - 	componentID : 6000
  WARN root [2019-09-10 16:41:45,093] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 16:41:46,597] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15100 ----
  INFO root [2019-09-10 16:41:46,597] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 16:41:46,598] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 16:41:46,598] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 16:41:47,001] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 16:41:47,008] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 16:41:54,200] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 16:41:54,298] - Components::findComponents: found logger, addr:192.168.56.1:63448
  INFO root [2019-09-10 16:41:54,298] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:41:54,299] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:41:54,300] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 16:41:54,402] - Components::findComponents: found dbmgr, addr:192.168.56.1:63447
  INFO root [2019-09-10 16:41:54,403] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:41:54,403] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:41:54,408] - Components::findComponents: found dbmgr, addr:192.168.56.1:63447
  WARN root [2019-09-10 16:41:55,408] - Forwarding logs to logger[192.168.56.1:63448/0/0/0]...
 ERROR root [2019-09-10 16:44:30,642] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=232784517058, lastReceivedTime=232749662014, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 16:44:30,643] - The following logs sent to logger failed:
  WARN root [2019-09-10 16:44:30,643] - ==> [2019-09-10 16:44:30 641] TCPPacketReceiver::processPendingEvents(192.168.56.1:63448): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:44:30,644] - ==> [2019-09-10 16:44:30 643] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=232784517058, lastReceivedTime=232749662014, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 16:44:30,739] - TCPPacketReceiver::processPendingEvents(192.168.56.1:63447): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:44:30,741] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=232784740568, lastReceivedTime=232749681480, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 16:44:30,747] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 16:44:30,749] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 16:44:30,749] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 16:44:30,749] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 16:48:24,917] - 
  INFO root [2019-09-10 16:48:24,924] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 16:48:24,925] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 16:48:24,925] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 16:48:24,925] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 16:48:24,965] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63553 )
  INFO root [2019-09-10 16:48:24,965] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63553, SOMAXCONN=5.
  INFO root [2019-09-10 16:48:24,965] - server-configs:
  INFO root [2019-09-10 16:48:24,965] - 	internalTcpAddr : 192.168.56.1:63553
  INFO root [2019-09-10 16:48:24,965] - 	componentID : 6000
  WARN root [2019-09-10 16:48:24,972] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-10 16:48:25,312] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20141, errorno=10048
  INFO root [2019-09-10 16:48:25,316] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 10032 ----
  INFO root [2019-09-10 16:48:25,316] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 16:48:25,316] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 16:48:25,317] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 16:48:25,718] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 16:48:25,719] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 16:48:32,847] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 16:48:32,891] - Components::findComponents: found logger, addr:192.168.56.1:63559
  INFO root [2019-09-10 16:48:32,892] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:48:32,892] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:48:32,893] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 16:48:32,995] - Components::findComponents: found dbmgr, addr:192.168.56.1:63555
  INFO root [2019-09-10 16:48:32,996] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:48:32,996] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:48:32,996] - Components::findComponents: found dbmgr, addr:192.168.56.1:63555
  WARN root [2019-09-10 16:48:33,996] - Forwarding logs to logger[192.168.56.1:63559/0/0/0]...
 ERROR root [2019-09-10 16:49:42,242] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=233485618029, lastReceivedTime=233446840713, inactivityExceptionPeriod=134886717)
  INFO root [2019-09-10 16:49:42,245] - The following logs sent to logger failed:
  WARN root [2019-09-10 16:49:42,246] - ==> [2019-09-10 16:49:42 235] TCPPacketReceiver::processPendingEvents(192.168.56.1:63559): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:49:42,246] - ==> [2019-09-10 16:49:42 243] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=233485618029, lastReceivedTime=233446840713, inactivityExceptionPeriod=134886717)
  WARN root [2019-09-10 16:49:42,336] - TCPPacketReceiver::processPendingEvents(192.168.56.1:63555): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:49:42,342] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=233485842036, lastReceivedTime=233446791676, inactivityExceptionPeriod=134886717)
  INFO root [2019-09-10 16:49:42,349] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 16:49:42,362] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 16:49:42,362] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 16:49:42,363] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 16:53:56,199] - 
  INFO root [2019-09-10 16:53:56,200] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 16:53:56,201] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 16:53:56,201] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 16:53:56,202] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 16:53:56,247] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63632 )
  INFO root [2019-09-10 16:53:56,247] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63632, SOMAXCONN=5.
  INFO root [2019-09-10 16:53:56,247] - server-configs:
  INFO root [2019-09-10 16:53:56,247] - 	internalTcpAddr : 192.168.56.1:63632
  INFO root [2019-09-10 16:53:56,247] - 	componentID : 6000
  WARN root [2019-09-10 16:53:56,250] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 16:53:57,756] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12160 ----
  INFO root [2019-09-10 16:53:57,765] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 16:53:57,765] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 16:53:57,766] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 16:53:58,161] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 16:53:58,161] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 16:54:05,304] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 16:54:05,339] - Components::findComponents: found logger, addr:192.168.56.1:63634
  INFO root [2019-09-10 16:54:05,340] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:54:05,340] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:54:05,341] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 16:54:05,442] - Components::findComponents: found dbmgr, addr:192.168.56.1:63633
  INFO root [2019-09-10 16:54:05,442] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:54:05,443] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:54:05,443] - Components::findComponents: found dbmgr, addr:192.168.56.1:63633
  WARN root [2019-09-10 16:54:06,444] - Forwarding logs to logger[192.168.56.1:63634/0/0/0]...
 ERROR root [2019-09-10 16:55:45,699] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=234303396362, lastReceivedTime=234259627831, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 16:55:45,702] - The following logs sent to logger failed:
  WARN root [2019-09-10 16:55:45,703] - ==> [2019-09-10 16:55:45 696] TCPPacketReceiver::processPendingEvents(192.168.56.1:63634): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:55:45,703] - ==> [2019-09-10 16:55:45 700] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=234303396362, lastReceivedTime=234259627831, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 16:55:45,809] - TCPPacketReceiver::processPendingEvents(192.168.56.1:63633): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:55:45,810] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=234303645958, lastReceivedTime=234259639864, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 16:55:45,815] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 16:55:45,818] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 16:55:45,819] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 16:55:45,820] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 16:58:49,889] - 
  INFO root [2019-09-10 16:58:49,890] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 16:58:49,892] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 16:58:49,892] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 16:58:49,892] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 16:58:49,917] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63709 )
  INFO root [2019-09-10 16:58:49,917] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63709, SOMAXCONN=5.
  INFO root [2019-09-10 16:58:49,917] - server-configs:
  INFO root [2019-09-10 16:58:49,917] - 	internalTcpAddr : 192.168.56.1:63709
  INFO root [2019-09-10 16:58:49,917] - 	componentID : 6000
  WARN root [2019-09-10 16:58:49,923] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 16:58:51,435] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12876 ----
  INFO root [2019-09-10 16:58:51,436] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 16:58:51,436] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 16:58:51,437] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 16:58:51,832] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 16:58:51,833] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 16:58:58,978] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 16:58:58,989] - Components::findComponents: found logger, addr:192.168.56.1:63708
  INFO root [2019-09-10 16:58:58,989] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:58:58,989] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:58:58,990] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-10 16:59:00,093] - Forwarding logs to logger[192.168.56.1:63708/0/0/0]...
 ERROR root [2019-09-10 16:59:47,029] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=234846388913, lastReceivedTime=234786550128, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 16:59:47,033] - The following logs sent to logger failed:
  WARN root [2019-09-10 16:59:47,033] - ==> [2019-09-10 16:59:47 022] TCPPacketReceiver::processPendingEvents(192.168.56.1:63708): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 16:59:47,034] - ==> [2019-09-10 16:59:47 031] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=234846388913, lastReceivedTime=234786550128, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 16:59:49,053] - 
  INFO root [2019-09-10 16:59:49,054] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 16:59:49,056] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 16:59:49,057] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 16:59:49,057] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 16:59:49,202] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63742 )
  INFO root [2019-09-10 16:59:49,203] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63742, SOMAXCONN=5.
  INFO root [2019-09-10 16:59:49,203] - server-configs:
  INFO root [2019-09-10 16:59:49,203] - 	internalTcpAddr : 192.168.56.1:63742
  INFO root [2019-09-10 16:59:49,203] - 	componentID : 6000
  WARN root [2019-09-10 16:59:49,215] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 16:59:49,685] - Components::findLogger: found logger, addr:192.168.56.1:63738
  INFO root [2019-09-10 16:59:49,685] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 16:59:49,685] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 16:59:49,699] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 5388 ----
  INFO root [2019-09-10 16:59:49,700] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 16:59:49,701] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 16:59:49,702] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 16:59:50,331] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 16:59:50,332] - Components::process(): Request for the process of identity...
  WARN root [2019-09-10 16:59:56,523] - Forwarding logs to logger[192.168.56.1:63738/0/0/0]...
 ERROR root [2019-09-10 17:00:08,963] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=234895739736, lastReceivedTime=234852386770, inactivityExceptionPeriod=134888460)
  INFO root [2019-09-10 17:00:08,968] - The following logs sent to logger failed:
  WARN root [2019-09-10 17:00:08,968] - ==> [2019-09-10 17:00:08 961] TCPPacketReceiver::processPendingEvents(192.168.56.1:63738): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:00:08,969] - ==> [2019-09-10 17:00:08 964] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=234895739736, lastReceivedTime=234852386770, inactivityExceptionPeriod=134888460)
  INFO root [2019-09-10 17:00:10,837] - 
  INFO root [2019-09-10 17:00:10,838] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 17:00:10,839] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 17:00:10,839] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 17:00:10,840] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 17:00:10,869] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63762 )
  INFO root [2019-09-10 17:00:10,869] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63762, SOMAXCONN=5.
  INFO root [2019-09-10 17:00:10,869] - server-configs:
  INFO root [2019-09-10 17:00:10,869] - 	internalTcpAddr : 192.168.56.1:63762
  INFO root [2019-09-10 17:00:10,869] - 	componentID : 6000
  WARN root [2019-09-10 17:00:10,870] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 17:00:12,376] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 348 ----
  INFO root [2019-09-10 17:00:12,377] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 17:00:12,378] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 17:00:12,378] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 17:00:12,811] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 17:00:12,811] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 17:00:19,959] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 17:00:19,981] - Components::findComponents: found logger, addr:192.168.56.1:63763
  INFO root [2019-09-10 17:00:19,982] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:00:19,982] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:00:19,983] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 17:01:18,591] - 
  INFO root [2019-09-10 17:01:18,592] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 17:01:18,593] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 17:01:18,594] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 17:01:18,594] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 17:01:18,640] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63784 )
  INFO root [2019-09-10 17:01:18,641] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63784, SOMAXCONN=5.
  INFO root [2019-09-10 17:01:18,641] - server-configs:
  INFO root [2019-09-10 17:01:18,641] - 	internalTcpAddr : 192.168.56.1:63784
  INFO root [2019-09-10 17:01:18,641] - 	componentID : 6000
  WARN root [2019-09-10 17:01:18,642] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 17:01:20,147] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13376 ----
  INFO root [2019-09-10 17:01:20,162] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 17:01:20,167] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 17:01:20,168] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 17:01:20,693] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 17:01:20,693] - Components::process(): Request for the process of identity...
  WARN root [2019-09-10 17:01:26,814] - TCPPacketReceiver::processPendingEvents(192.168.56.1:63808): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-10 17:01:27,830] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 17:01:27,920] - Components::findComponents: found logger, addr:192.168.56.1:63785
  INFO root [2019-09-10 17:01:27,921] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:01:27,921] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:01:27,923] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 17:01:28,025] - Components::findComponents: found dbmgr, addr:192.168.56.1:63787
  INFO root [2019-09-10 17:01:28,026] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:01:28,026] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:01:28,026] - Components::findComponents: found dbmgr, addr:192.168.56.1:63787
  WARN root [2019-09-10 17:01:29,027] - Forwarding logs to logger[192.168.56.1:63785/0/0/0]...
 ERROR root [2019-09-10 17:03:26,794] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235340861480, lastReceivedTime=235322553388, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 17:03:26,803] - The following logs sent to logger failed:
  WARN root [2019-09-10 17:03:26,804] - ==> [2019-09-10 17:03:26 793] TCPPacketReceiver::processPendingEvents(192.168.56.1:63785): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:03:26,804] - ==> [2019-09-10 17:03:26 795] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235340861480, lastReceivedTime=235322553388, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 17:03:26,905] - TCPPacketReceiver::processPendingEvents(192.168.56.1:63787): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:03:26,910] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235341121895, lastReceivedTime=235322651096, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 17:03:26,913] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 17:03:26,914] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 17:03:26,914] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 17:03:26,915] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 17:03:28,635] - 
  INFO root [2019-09-10 17:03:28,635] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 17:03:28,636] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 17:03:28,637] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 17:03:28,637] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 17:03:28,655] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63867 )
  INFO root [2019-09-10 17:03:28,656] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63867, SOMAXCONN=5.
  INFO root [2019-09-10 17:03:28,656] - server-configs:
  INFO root [2019-09-10 17:03:28,656] - 	internalTcpAddr : 192.168.56.1:63867
  INFO root [2019-09-10 17:03:28,656] - 	componentID : 6000
  WARN root [2019-09-10 17:03:28,656] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 17:03:30,160] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 14392 ----
  INFO root [2019-09-10 17:03:30,161] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 17:03:30,161] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 17:03:30,161] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 17:03:30,549] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 17:03:30,549] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 17:03:37,683] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 17:03:37,752] - Components::findComponents: found logger, addr:192.168.56.1:63866
  INFO root [2019-09-10 17:03:37,753] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:03:37,753] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:03:37,754] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 17:03:37,857] - Components::findComponents: found dbmgr, addr:192.168.56.1:63864
  INFO root [2019-09-10 17:03:37,857] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:03:37,857] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:03:37,857] - Components::findComponents: found dbmgr, addr:192.168.56.1:63864
  WARN root [2019-09-10 17:03:38,857] - Forwarding logs to logger[192.168.56.1:63866/0/0/0]...
 ERROR root [2019-09-10 17:04:46,584] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235520385681, lastReceivedTime=235479969436, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 17:04:46,585] - The following logs sent to logger failed:
  WARN root [2019-09-10 17:04:46,585] - ==> [2019-09-10 17:04:46 583] TCPPacketReceiver::processPendingEvents(192.168.56.1:63866): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:04:46,585] - ==> [2019-09-10 17:04:46 584] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235520385681, lastReceivedTime=235479969436, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-10 17:04:46,684] - TCPPacketReceiver::processPendingEvents(192.168.56.1:63864): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:04:46,690] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235520627537, lastReceivedTime=235479878819, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 17:04:46,696] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 17:04:46,697] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 17:04:46,697] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 17:04:46,697] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 17:04:48,393] - 
  INFO root [2019-09-10 17:04:48,398] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 17:04:48,398] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 17:04:48,399] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 17:04:48,400] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 17:04:48,427] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63906 )
  INFO root [2019-09-10 17:04:48,427] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63906, SOMAXCONN=5.
  INFO root [2019-09-10 17:04:48,427] - server-configs:
  INFO root [2019-09-10 17:04:48,427] - 	internalTcpAddr : 192.168.56.1:63906
  INFO root [2019-09-10 17:04:48,428] - 	componentID : 6000
  WARN root [2019-09-10 17:04:48,428] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 17:04:49,933] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 7420 ----
  INFO root [2019-09-10 17:04:49,934] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 17:04:49,934] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 17:04:49,935] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 17:04:50,358] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 17:04:50,358] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 17:04:57,476] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 17:04:57,484] - Components::findComponents: found logger, addr:192.168.56.1:63908
  INFO root [2019-09-10 17:04:57,484] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:04:57,485] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:04:57,486] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-10 17:04:58,591] - Forwarding logs to logger[192.168.56.1:63908/0/0/0]...
 ERROR root [2019-09-10 17:05:24,849] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235606484519, lastReceivedTime=235592845201, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 17:05:24,850] - The following logs sent to logger failed:
  WARN root [2019-09-10 17:05:24,851] - ==> [2019-09-10 17:05:24 849] TCPPacketReceiver::processPendingEvents(192.168.56.1:63908): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:05:24,851] - ==> [2019-09-10 17:05:24 850] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235606484519, lastReceivedTime=235592845201, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 17:05:26,560] - 
  INFO root [2019-09-10 17:05:26,566] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 17:05:26,567] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 17:05:26,568] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 17:05:26,568] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 17:05:26,610] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63926 )
  INFO root [2019-09-10 17:05:26,610] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63926, SOMAXCONN=5.
  INFO root [2019-09-10 17:05:26,611] - server-configs:
  INFO root [2019-09-10 17:05:26,611] - 	internalTcpAddr : 192.168.56.1:63926
  INFO root [2019-09-10 17:05:26,611] - 	componentID : 6000
  WARN root [2019-09-10 17:05:26,613] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 17:05:28,131] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 248 ----
  INFO root [2019-09-10 17:05:28,132] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 17:05:28,132] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 17:05:28,133] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 17:05:28,517] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 17:05:28,517] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 17:05:35,653] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 17:05:35,718] - Components::findComponents: found logger, addr:192.168.56.1:63927
  INFO root [2019-09-10 17:05:35,718] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:05:35,718] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:05:35,719] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-10 17:05:36,821] - Forwarding logs to logger[192.168.56.1:63927/0/0/0]...
 ERROR root [2019-09-10 17:05:51,200] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235665773368, lastReceivedTime=235630940990, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-10 17:05:51,202] - The following logs sent to logger failed:
  WARN root [2019-09-10 17:05:51,202] - ==> [2019-09-10 17:05:51 200] TCPPacketReceiver::processPendingEvents(192.168.56.1:63927): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:05:51,202] - ==> [2019-09-10 17:05:51 201] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235665773368, lastReceivedTime=235630940990, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-10 17:05:52,995] - 
  INFO root [2019-09-10 17:05:53,002] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 17:05:53,003] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 17:05:53,003] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 17:05:53,003] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 17:05:53,046] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:63945 )
  INFO root [2019-09-10 17:05:53,046] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:63945, SOMAXCONN=5.
  INFO root [2019-09-10 17:05:53,046] - server-configs:
  INFO root [2019-09-10 17:05:53,046] - 	internalTcpAddr : 192.168.56.1:63945
  INFO root [2019-09-10 17:05:53,046] - 	componentID : 6000
  WARN root [2019-09-10 17:05:53,051] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 17:05:54,567] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 8464 ----
  INFO root [2019-09-10 17:05:54,568] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 17:05:54,568] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 17:05:54,568] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 17:05:54,966] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 17:05:54,974] - Components::process(): Request for the process of identity...
  INFO root [2019-09-10 17:06:02,091] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 17:06:02,104] - Components::findComponents: found logger, addr:192.168.56.1:63948
  INFO root [2019-09-10 17:06:02,104] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:06:02,105] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:06:02,106] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 17:06:02,209] - Components::findComponents: found dbmgr, addr:192.168.56.1:63941
  INFO root [2019-09-10 17:06:02,209] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:06:02,209] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:06:02,209] - Components::findComponents: found dbmgr, addr:192.168.56.1:63941
  WARN root [2019-09-10 17:06:03,211] - Forwarding logs to logger[192.168.56.1:63948/0/0/0]...
 ERROR root [2019-09-10 17:07:46,244] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235924621933, lastReceivedTime=235872508522, inactivityExceptionPeriod=134889067)
  INFO root [2019-09-10 17:07:46,247] - The following logs sent to logger failed:
  WARN root [2019-09-10 17:07:46,248] - ==> [2019-09-10 17:07:46 243] TCPPacketReceiver::processPendingEvents(192.168.56.1:63948): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:07:46,248] - ==> [2019-09-10 17:07:46 245] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235924621933, lastReceivedTime=235872508522, inactivityExceptionPeriod=134889067)
  WARN root [2019-09-10 17:07:46,353] - TCPPacketReceiver::processPendingEvents(192.168.56.1:63941): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:07:46,353] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=235924868288, lastReceivedTime=235872235933, inactivityExceptionPeriod=134889067)
  INFO root [2019-09-10 17:07:46,356] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 17:07:46,368] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 17:07:46,369] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 17:07:46,369] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 17:07:48,147] - 
  INFO root [2019-09-10 17:07:48,152] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 17:07:48,153] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 17:07:48,153] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 17:07:48,153] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 17:07:48,212] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64011 )
  INFO root [2019-09-10 17:07:48,213] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64011, SOMAXCONN=5.
  INFO root [2019-09-10 17:07:48,213] - server-configs:
  INFO root [2019-09-10 17:07:48,213] - 	internalTcpAddr : 192.168.56.1:64011
  INFO root [2019-09-10 17:07:48,213] - 	componentID : 6000
  WARN root [2019-09-10 17:07:48,214] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 17:07:48,595] - Components::findLogger: found logger, addr:192.168.56.1:64007
  INFO root [2019-09-10 17:07:48,595] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 17:07:48,595] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 17:07:48,600] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13744 ----
  INFO root [2019-09-10 17:07:48,600] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 17:07:48,600] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 17:07:48,600] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 17:07:49,029] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 17:07:49,030] - Components::process(): Request for the process of identity...
  WARN root [2019-09-10 17:07:55,147] - Forwarding logs to logger[192.168.56.1:64007/0/0/0]...
 ERROR root [2019-09-10 17:10:01,131] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=236228120052, lastReceivedTime=236198843258, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 17:10:01,133] - The following logs sent to logger failed:
  WARN root [2019-09-10 17:10:01,134] - ==> [2019-09-10 17:10:01 130] TCPPacketReceiver::processPendingEvents(192.168.56.1:64007): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:10:01,134] - ==> [2019-09-10 17:10:01 132] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=236228120052, lastReceivedTime=236198843258, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 17:10:01,229] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64008): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 17:10:01,232] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=236228344805, lastReceivedTime=236198937578, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 17:10:01,247] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 17:10:01,249] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 17:10:01,250] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 17:10:01,250] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 19:19:39,845] - 
  INFO root [2019-09-10 19:19:39,882] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 19:19:39,882] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 19:19:39,882] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 19:19:39,882] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 19:19:39,966] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64276 )
  INFO root [2019-09-10 19:19:39,966] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64276, SOMAXCONN=5.
  INFO root [2019-09-10 19:19:39,966] - server-configs:
  INFO root [2019-09-10 19:19:39,966] - 	internalTcpAddr : 192.168.56.1:64276
  INFO root [2019-09-10 19:19:39,966] - 	componentID : 6000
  WARN root [2019-09-10 19:19:39,966] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 19:19:41,482] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 3844 ----
  INFO root [2019-09-10 19:19:41,486] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 19:19:41,486] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 19:19:41,486] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 19:19:41,895] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 19:19:41,895] - Components::process(): Request for the process of identity...
  WARN root [2019-09-10 19:19:48,151] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64296): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-10 19:19:49,171] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 19:19:49,270] - Components::findComponents: found logger, addr:192.168.56.1:64280
  INFO root [2019-09-10 19:19:49,270] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 19:19:49,270] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 19:19:49,270] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 19:19:49,391] - Components::findComponents: found dbmgr, addr:192.168.56.1:64275
  INFO root [2019-09-10 19:19:49,391] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 19:19:49,391] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 19:19:49,391] - Components::findComponents: found dbmgr, addr:192.168.56.1:64275
  WARN root [2019-09-10 19:19:50,401] - Forwarding logs to logger[192.168.56.1:64280/0/0/0]...
 ERROR root [2019-09-10 19:21:39,651] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=254010135445, lastReceivedTime=253943365443, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 19:21:39,661] - The following logs sent to logger failed:
  WARN root [2019-09-10 19:21:39,661] - ==> [2019-09-10 19:21:39 659] TCPPacketReceiver::processPendingEvents(192.168.56.1:64280): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 19:21:39,661] - ==> [2019-09-10 19:21:39 661] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=254010135445, lastReceivedTime=253943365443, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-10 19:21:39,741] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64275): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 19:21:39,751] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=254010343073, lastReceivedTime=253943366481, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 19:21:39,755] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 19:21:39,755] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 19:21:39,755] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 19:21:39,755] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 19:21:41,329] - 
  INFO root [2019-09-10 19:21:41,333] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 19:21:41,335] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 19:21:41,335] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 19:21:41,335] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 19:21:41,352] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64355 )
  INFO root [2019-09-10 19:21:41,352] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64355, SOMAXCONN=5.
  INFO root [2019-09-10 19:21:41,352] - server-configs:
  INFO root [2019-09-10 19:21:41,352] - 	internalTcpAddr : 192.168.56.1:64355
  INFO root [2019-09-10 19:21:41,352] - 	componentID : 6000
  WARN root [2019-09-10 19:21:41,352] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 19:21:42,871] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12012 ----
  INFO root [2019-09-10 19:21:42,871] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 19:21:42,871] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 19:21:42,871] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 19:21:43,292] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 19:21:43,296] - Components::process(): Request for the process of identity...
  WARN root [2019-09-10 19:21:43,985] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20522, errorno=10048
  INFO root [2019-09-10 19:21:51,231] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 19:21:51,251] - Components::findComponents: found logger, addr:192.168.56.1:64352
  INFO root [2019-09-10 19:21:51,251] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 19:21:51,251] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 19:21:51,251] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 19:21:51,261] - Components::findComponents: found logger, addr:192.168.56.1:64352
  INFO root [2019-09-10 19:21:51,261] - Components::findComponents: found dbmgr, addr:192.168.56.1:64351
  INFO root [2019-09-10 19:21:51,261] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 19:21:51,261] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 19:21:51,371] - Components::findComponents: found dbmgr, addr:192.168.56.1:64351
  WARN root [2019-09-10 19:21:52,372] - Forwarding logs to logger[192.168.56.1:64352/0/0/0]...
 ERROR root [2019-09-10 19:26:38,286] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=254682042795, lastReceivedTime=254621345798, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 19:26:38,288] - The following logs sent to logger failed:
  WARN root [2019-09-10 19:26:38,288] - ==> [2019-09-10 19:26:38 281] TCPPacketReceiver::processPendingEvents(192.168.56.1:64352): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 19:26:38,288] - ==> [2019-09-10 19:26:38 286] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=254682042795, lastReceivedTime=254621345798, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-10 19:26:38,384] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64351): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 19:26:38,388] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=254682274956, lastReceivedTime=254621295547, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-10 19:26:38,392] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 19:26:38,392] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 19:26:38,392] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 19:26:38,393] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-10 19:26:40,192] - 
  INFO root [2019-09-10 19:26:40,209] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-10 19:26:40,210] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-10 19:26:40,210] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-10 19:26:40,210] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-10 19:26:40,296] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64464 )
  INFO root [2019-09-10 19:26:40,297] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64464, SOMAXCONN=5.
  INFO root [2019-09-10 19:26:40,297] - server-configs:
  INFO root [2019-09-10 19:26:40,297] - 	internalTcpAddr : 192.168.56.1:64464
  INFO root [2019-09-10 19:26:40,297] - 	componentID : 6000
  WARN root [2019-09-10 19:26:40,298] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-10 19:26:41,804] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13592 ----
  INFO root [2019-09-10 19:26:41,804] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-10 19:26:41,805] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-10 19:26:41,805] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-10 19:26:42,246] - ---- cellappmgr is running ----
 DEBUG root [2019-09-10 19:26:42,247] - Components::process(): Request for the process of identity...
  WARN root [2019-09-10 19:26:48,401] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64478): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-10 19:26:49,407] - Components::findComponents: find logger(1)...
  INFO root [2019-09-10 19:26:49,500] - Components::findComponents: found logger, addr:192.168.56.1:64465
  INFO root [2019-09-10 19:26:49,500] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 19:26:49,501] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 19:26:49,504] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-10 19:26:49,608] - Components::findComponents: found dbmgr, addr:192.168.56.1:64459
  INFO root [2019-09-10 19:26:49,608] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-10 19:26:49,609] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-10 19:26:49,609] - Components::findComponents: found dbmgr, addr:192.168.56.1:64459
  WARN root [2019-09-10 19:26:50,609] - Forwarding logs to logger[192.168.56.1:64465/0/0/0]...
 ERROR root [2019-09-10 19:29:19,214] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=255044132542, lastReceivedTime=255024067871, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 19:29:19,220] - The following logs sent to logger failed:
  WARN root [2019-09-10 19:29:19,220] - ==> [2019-09-10 19:29:19 213] TCPPacketReceiver::processPendingEvents(192.168.56.1:64465): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 19:29:19,221] - ==> [2019-09-10 19:29:19 215] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=255044132542, lastReceivedTime=255024067871, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-10 19:29:19,309] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64459): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-10 19:29:19,310] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=255044349490, lastReceivedTime=255023828489, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-10 19:29:19,314] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-10 19:29:19,314] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-10 19:29:19,314] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-10 19:29:19,314] - ThreadPool::destroy(): starting size 2.
